src/base/static/js/routes.js

```diff
@@ -156,12 +157,21 @@ var Shareabouts = Shareabouts || {};
       this.appView.newPlace();
     },
 
-    viewLandmark: function(id) {
-      this.appView.viewLandmark(id, { zoom: this.loading });
+    viewLandmark: function(modelId, responseId) {
```

Comment:
We can probably refactor `viewLandmark` and `viewPlace` below into a single handler function that calls `viewPlaceOrLandmark`. Maybe we can call this function `handlePlaceOrLandmark`